package.json

```diff
@@ -1,47 +1,71 @@
   "dependencies": {
-    "node": "^14.15.0",
-    "npm": "^6.9.0"
+    "@commitlint/config-conventional": "^17.2.0",
+    "@salesforce/core": "^3.23.3",
+    "@types/istanbul-reports": "^3.0.1",
+    "commitlint": "^17.2.0",
+    "faye": "1.4.0",
+    "glob": "^8.0.3",
+    "istanbul-lib-coverage": "^3.2.0",
+    "istanbul-lib-report": "^3.0.0",
+    "istanbul-reports": "^3.1.4"
   },
   "devDependencies": {
-    "@commitlint/cli": "^7",
-    "@commitlint/config-conventional": "^7",
-    "@typescript-eslint/eslint-plugin": "^2.22.0",
-    "@types/mocha": "^9",
-    "@typescript-eslint/parser": "^2.22.0",
-    "commitizen": "^3.0.5",
-    "cz-conventional-changelog": "^2.1.0",
-    "eslint": "^6.8.0",
+    "@salesforce/ts-sinon": "^1.4.2",
+    "@salesforce/ts-types": "1.2.2",
+    "@types/chai": "^4",
+    "@types/glob": "^8.0.0",
+    "@types/istanbul-lib-coverage": "^2.0.4",
+    "@types/istanbul-lib-report": "^3.0.0",
+    "@types/mocha": "^10",
+    "@types/node": "^14",
+    "@types/sinon": "^7.5.2",
+    "@typescript-eslint/eslint-plugin": "^5.42.1",
+    "@typescript-eslint/parser": "^5.42.1",
+    "chai": "^4",
+    "cz-conventional-changelog": "^3.3.0",
+    "eslint": "^7.1.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-header": "^3.0.0",
     "eslint-plugin-jsdoc": "^19.1.0",
     "eslint-plugin-prettier": "^3.1.4",
     "husky": "^4.2.5",
-    "lerna": "^3.20.2",
     "lint-staged": "^10.2.11",
     "mocha": "^10",
+    "mocha-junit-reporter": "^1.23.3",
+    "nyc": "^14.1.1",
     "prettier": "1.19.1",
+    "shx": "^0.3.4",
+    "sinon": "^7.3.1",
+    "source-map-support": "^0.5.16",
+    "ts-node": "^10.7.0",
     "typescript": "4.6.4"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/forcedotcom/salesforcedx-apex.git"
-  },
-  "license": "BSD-3-Clause",
+  "homepage": "https://github.com/forcedotcom/salesforcedx-apex",
+  "keywords": [
+    "apex",
+    "force",
+    "salesforce",
+    "sfdx",
+    "salesforcedx"
+  ],
+  "files": [
+    "lib/src"
+  ],
   "scripts": {
-    "build": "lerna run build",
-    "bump-versions": "lerna version --force-publish --no-git-tag-version --exact --yes",
+    "build": "shx rm -rf lib && tsc -b",
     "commit-init": "commitizen init cz-conventional-changelog --save-dev --save-exact --force",
     "commit": "git-cz",
-    "format": "lerna run format",
-    "lint": "lerna run lint",
-    "manifest:generate": "lerna run manifest:generate",
-    "plugin:link": "cd packages/plugin-apex && sfdx plugins:link && cd ../.. && yarn install && yarn build",
-    "plugin:unlink": "cd packages/plugin-apex && sfdx plugins:unlink",
-    "postinstall": "lerna bootstrap --no-ci",
-    "test": "lerna run test",
-    "package": "lerna run package"
+    "format": "prettier --config ./.prettierrc --write './{src,test,scripts}/**/*.{ts,js,json}'",
+    "lint": "eslint -c .eslintrc.json --ext .ts ./src ./test",
+    "test": "nyc mocha ./test/**/*.test.ts",
```

Comment:
I moved the matching pattern here and removed it from mocharc to get the "Run Current Test" launch config to work.  When the matching pattern is in the mocharc file, it always supercedes whatever arg is passed to mocha, which doesn't allow Run Current Test to work correctly.  See https://github.com/mochajs/mocha/issues/1444